README.md

```diff
@@ -1,108 +1,135 @@
 
-When working on `RHOAM`, all `make` commands should be prefixed with `INSTALLATION_TYPE=managed-api`.
-This will configure any variables that may be required.
+An Integreatly Operator can be installed using two different flavours: `managed` or `managed-api`
 
-The default `INSTALLATION_TYPE` is `managed` which configures variables for use with `RHMI`
+To switch between the two you can use export the `INSTALLATION_TYPE` env or use it in conjunction with any of the make commands referenced in this README
 
----
+### Installed products
 
-A Kubernetes Operator based on the Operator SDK for installing and reconciling Integreatly products.
+The operator installs the following products:
+
+### managed
 
-### Installed products
-Currently the operator installs the following products:
 - AMQ Online
 - AMQ Streams
 - Codeready
 - Fuse
 - Nexus
-- RHSSO (both a cluster instance and a user instance)
+- RHSSO (both a cluster instance, and a user instance)
 - 3scale
 - Integreatly solution explorer
 
-## Prerequisites
+### managed-api
+
+- 3scale
+- RHSSO (both a cluster instance, and a user instance)
+- Marin3r
+
 
-NOTE: Due to a change in how networking is configured for openshift in v4.4.6 (mentioned in the [cloud resource operator](https://github.com/integr8ly/cloud-resource-operator#supported-openshift-versions)) there is a limitation on the version of Openshift that RHMI can be installed on for BYOC clusters.
-Due to this change the use of integreatly-operator <= v2.4.0 on Openshift >= v4.4.6 is unsupported. Please use >= v2.5.0 of integreatly-operator for Openshift >= v4.4.6.
+## Prerequisites
 
-- [operator-sdk](https://github.com/operator-framework/operator-sdk) version v0.15.1.
+- [operator-sdk](https://github.com/operator-framework/operator-sdk) version v1.2.0.
 - [go](https://golang.org/dl/) version 1.13.4+
 - [moq](https://github.com/matryer/moq)
-- [oc](https://docs.okd.io/3.11/cli_reference/get_started_cli.html#cli-reference-get-started-cli) version v3.11+
-- Access to an Openshift v4.2.0+ cluster
+- [oc](https://docs.okd.io/latest/cli_reference/openshift_cli/getting-started-cli.html) version v4.6+
+- Access to an Openshift v4.6.0+ cluster
 - A user with administrative privileges in the OpenShift cluster
-- AWS account with permissions to create S3 buckets
 
-After installation, the following commands must be run to avoid a known [issue](https://github.com/matryer/moq/issues/98) related to the Moq package:
-```
+After installation, the following commands must be run to avoid a known issue related to the Moq package:
+```shell
 make code/compile
 go install github.com/matryer/moq
 ```
 
-## Local Setup
+## Using `ocm` for installation of RHMI
+
+If you want to test your changes on a cluster, the easiest solution would be to spin up OSD 4 cluster using `ocm`.
+See [here](https://github.com/integr8ly/delorean/tree/master/docs/ocm) for an up to date guide on how to do this.
```

Comment:
Would it be worth mentioning how much resources are needed on the cluster in order to install RHOAM? Just in order to prevent failed installs from lack of resources

Reply:
Yes, we can do that. I would keep it pretty simple though, just mention the vcpu and if you have a smaller cluster to use IN_PROW=true variable mentioned in the table in order to get a smaller installation. 👍